openscap_report/html_report/templates/js/interactive_script.js

```diff
@@ -43,9 +43,11 @@ function get_next_five_rules() {
     const next_five_rules = [];
     var count_five = 5;
     for(let i = 0; i <= count_five && FIRST_HIDDEN_ELEMENT !== null; i += 1) {
-        next_five_rules.push(FIRST_HIDDEN_ELEMENT);
+        if (FIRST_HIDDEN_ELEMENT !== null) {
+            next_five_rules.push(FIRST_HIDDEN_ELEMENT);
+        }
         FIRST_HIDDEN_ELEMENT = FIRST_HIDDEN_ELEMENT.nextSibling;
```

Comment:
If FIRST_HIDDEN_ELEMENT is null this won't work as well.

Reply:
Yes, it is true. But I didn't notice the condition in a loop. So this condition is not necessary.